containerfixture/containerfixture.go

```diff
@@ -0,0 +1,133 @@
+package containerfixture
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"text/template"
+	"time"
+)
+
+const waitTimeoutSeconds = int64(120)
```

Comment:
"wait timeout" is both redundant and opaque - try saying what the timeout is for.

Also you don't need to cast the constant - Go will do it for you when you use it.